docs/development/upgrading-node.md

```diff
@@ -70,49 +66,35 @@ We need to generate a patch file from each patch applied to V8.
-8. Update `patches/common/v8/README.md` with references to all new patches that have been added so that the next person will know which need to be removed.
-9. Update Electron's submodule references:
-  ```sh
-  $ cd electron/vendor/node
-  electron/vendor/node$ git fetch
-  electron/vendor/node$ git checkout electron-node-vA.B.C
-  electron/vendor/node$ cd ../libchromiumcontent
-  electron/vendor/libchromiumcontent$ git fetch
-  electron/vendor/libchromiumcontent$ git checkout upgrade-to-chromium-X
-  electron/vendor/libchromiumcontent$ cd ../..
-  electron$ git add vendor
-  electron$ git commit -m "update submodule references for node and libcc"
-  electron$ git push origin upgrade-to-chromium-<VERSION>
-  electron$ script/bootstrap.py -d
-  electron$ script/build.py -c -D
-  ```
+7. Update `patches/common/v8/README.md` with references to all new patches that have been added so that the next person will know which need to be removed.
+8. Update Electron's [DEPS](https://github.com/electron/electron/blob/master/DEPS#L5) with the commit hash of the latest commit in the `electron-node-vX.Y.Z` branch.
 
 ## Notes
 
-- libcc and V8 are treated as a single unit
 - Node maintains its own fork of V8
   - They backport a small amount of things as needed
   - Documentation in node about how [they work with V8](https://nodejs.org/api/v8.html)
 - We update code such that we only use one copy of V8 across all of electron
-  - E.g electron, libcc, and node
+  - E.g electron, chromium, and node
 - We don’t track upstream closely due to logistics:
    - Upstream uses multiple repos and so merging into a single repo
    would result in lost history. So we only update when we’re planning
    a node version bump in electron.
-- libcc is large and time-consuming to update, so we typically
+- Chromium is large and time-consuming to update, so we typically
   choose the node version based on which of its releases has a version
-  of V8 that’s closest to the version in libcc that we’re using.
+  of V8 that’s closest to the version in Chromium that we’re using.
   - We sometimes have to wait for the next periodic Node release
-   because it will sync more closely with the version of V8 in the new libcc
- - Electron keeps all its patches in libcc because it’s simpler than
+   because it will sync more closely with the version of V8 in the new Chromium
+ - Electron keeps all its patches in the repo because it’s simpler than
    maintaining different repos for patches for each upstream project.
-   - Crashpad, node, libcc, etc. patches are all kept in the same place
+   - Crashpad, node, chromium, skia etc. patches are all kept in the same place
  - Building node:
-   - There’s a chance we need to change our build configuration
-   to match the build flags that node wants in `node/common.gypi`
+   - We maintain our own GN build files for node.js to make it easier to ensure
+     that node, chromium and electron are all built with the same compiler flags.
```

Comment:
Node.js, Chromium, and Electron -- or maybe just "everything is built with the same compiler flags" :smile: